suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -4,8 +4,7 @@ import { memoizeWithArgs } from 'proxy-memoize';
 import {
     AccountsRootState,
     selectAccountByKey,
-    selectAccountTransactions,
-    selectTransactionByTxidAndAccountKey,
+    selectAccountTransactions as SelectCommonAccountTransactions,
```

Comment:
Why do you rename it and why it has capital letter?

Reply:
After Slack discussion renamed back in commit https://github.com/trezor/trezor-suite/pull/8169/commits/cfb6bf6323c20e644fed161a24c4ea5be437b47c.